snomed-database-loader/load_release-postgresql.sh

```diff
@@ -41,7 +41,20 @@ if [[ $1 == *uk_sct2mo* ]]; then
 	isMonolith=true
 fi
 
-databaseUri="postgresql://${PS_DB_OWNER_NAME}:${POSTGRES_PASSWORD}@${PS_DB_HOST}:${PS_DB_PORT}/${dbName}"
+stringencoder() {
+    # urlencode <string>
+    local length="${#1}"
+    for (( i = 0; i < length; i++ )); do
+        local c="${1:i:1}"
```

Comment:
What is `c` in this instance?

Reply:
Awesome! Could we possibly change `c` to `character` assuming this isn't a reserved word in bash - think it could be a helpful readability wise :D